src/clubb_driver.F90

```diff
@@ -988,7 +997,10 @@ subroutine run_clubb ( ngrdcol, calls_per_out, l_output_multi_col, l_output_doub
       sfc_elevation_nl, &
       p_sfc_nl, &
       T_sfc_nl, &
-      fcor_nl
+      fcor_nl ! obsolete
+      ! fcor_nl can be read from a namelist but is not used for any calculation.
+      ! fcor and fcory are calculated from lat_vals from a namelist.
```

Comment:
Is fcor always generated from lat_vals?  I think we want to retain the capability to directly specify fcor.

Reply:
Because of this change, we don't expect the results to be BFB.  We'll have to see how much the answers change.  If not much, then we could delete fcor_nl.  But if the answers do change, then maybe we do want to keep the option of specifying fcor, just so that we match our benchmark LES better.